src/app/concrete_dst_router.rs

```diff
@@ -0,0 +1,206 @@
+use futures::Poll;
+use http;
+use std::fmt;
+use std::marker::PhantomData;
+use std::time::Duration;
+
+use never::Never;
+
+use svc;
+use super::dst::DstAddr;
+use proxy::http::settings;
+use proxy::http::profiles::ConcreteDst;
+
+extern crate linkerd2_router as rt;
+
+pub use self::rt::{error, Router};
+
+#[allow(dead_code)]
+type Error = Box<dyn std::error::Error + Send + Sync>;
+
+#[derive(Clone, Debug)]
+pub struct Config {
```

Comment:
Nit: This looks like the same type as [`router::Config`](https://github.com/linkerd/linkerd2-proxy/blob/b70c68d4504a362eac6a7828039a2e5c7fcd308a/src/proxy/http/router.rs#L17-L21), maybe we should just reuse that?

Reply:
I don't think we should use the same type, as these routers are different (and will diverge more soon).  Specifically, why does this type need a capacity or max_idle_age? Don't we know the capacity from the dst overrides? We don't intend to idle these out, so a configurable max_idle_age is unnecessary...

Reply:
My understanding is that there will be a concrete dst router per logical dst and these routers will not get rebuilt when the service profile changes.  so I don't think the capacity can be known ahead of time.